openmrs/i18n/adt/locale_en.json

```diff
@@ -60,7 +60,7 @@
   "RELATIONSHIP_CAREGIVER": "Caregiver",
```

Comment:
@IipperMangesh Please remove RELATIONSHIP_DOCTOR, RELATIONSHIP_PATIENT, RELATIONSHIP_SUPERVISOR, RELATIONSHIP_SUPERVISEE, RELATIONSHIP_CAREGIVER from all translation files as they are not being used anymore.